src/Workspaces/Core/Portable/PatternMatching/PatternMatcher.cs

```diff
@@ -293,11 +136,34 @@ private static bool ContainsUpperCaseLetter(string pattern)
 
         private PatternMatch? MatchPatternChunk(
             string candidate,
-            bool includeMatchSpans,
             TextChunk patternChunk,
             bool punctuationStripped,
             bool fuzzyMatch)
         {
+            return fuzzyMatch
```

Comment:
💭 Well this is fascinating. The old code was performing the non-fuzzy match operation twice (with the second time always failing) before doing the fuzzy match.